### PR TITLE
Revert EventAttribute field types

### PR DESCRIPTION
**Pull request by LCyson**
## Describe your changes and provide context
This PR updates EventAttribute `key` and `value` data type. Please find details in this PR https://github.com/sei-protocol/sei-tendermint/pull/85

## Testing performed to validate your change
- unit test

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -145,7 +145,7 @@ replace (
 	github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.7.0
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
-	github.com/tendermint/tendermint => github.com/sei-protocol/sei-tendermint v0.1.171
+	github.com/tendermint/tendermint => github.com/sei-protocol/sei-tendermint v0.1.179-0.20230315064114-c903df251edf
```
Comment (BrandonWeng) [inline go.mod:148]: Was this intentional?

Comment (BrandonWeng) [reply go.mod:148]: Can we tag a new tendermint release instead

**Review by BrandonWeng — COMMENTED** [review]
Should be fine for parallelTx validation if it's just changing this. 

Could you also spin up a loadtest cluster just to make sure that it doesn't have any untended side effects before we deploy to sei-devnet-3?

File: go.mod (hunk 1 above)
Comment (philipsu522) [reply go.mod:148]: +1. Let's revert this and the protocol-team can cut a new release

**Review by philipsu522 — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (LCyson) [reply go.mod:148]: yes will update it to a tm release after this is merged https://github.com/sei-protocol/sei-tendermint/pull/85

**Pull request closed without merging**